pages/common/picom-trans.md

```diff
@@ -0,0 +1,20 @@
+# picom-trans
+
+> Set window opacity.
+> More information: <https://github.com/yshui/picom>.
+
+- Set the currently focused window opacity to a certain percentage:
+
+`picom-trans --current --opacity={{90}}`
+
+- Set the opacity of a window with a specific name:
```

Comment:
```suggestion
- Set an opacity of a window with a specific name:
```

Reply:
I think `the` is better in this context, because every window has one opacity. And we're setting this very opacity. `An opacity` sounds like we're setting one of the many opacities a window could have.